JS architecture update in NPM (#425)
rudderlabs/rudderstack-docs: stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/quick-start-guide.mdx

### Alternative installation using NPM

> It is recommended to use the snippets mentioned above to use the JavaScript SDK with your website. However, you can also use this [**NPM module**](https://www.npmjs.com/package/rudder-sdk-js) with the new JavaScript SDK v1.1 architecture to package RudderStack directly into your project.

Reviewer: ```suggestion
While it is recommended to use the above-mentioned snippets to use the JavaScript SDK with your website, you can also use the [**NPM module**](https://www.npmjs.com/package/rudder-sdk-js) with the new SDK(v1.1) architecture to package RudderStack directly into your project.
```

Reviewer: Should we also reference this somewhere: "Integrations will be loaded dynamically." cc @MoumitaM something along the lines of:

"...to page RudderStack directly into your project and load the integrations dynamically."

If the sentence is too long, maybe we can put the recommendation as a note and start the section with 

"You can also use the NPM module with the new SDK architecture to package RudderStack directly into your project and load the integrations dynamically."

> While it is recommended to use the above-mentioned snippets to use the JavaScript SDK with your website, you can also use the [**NPM module**](https://www.npmjs.com/package/rudder-sdk-js) with the new SDK(v1.1) architecture to package RudderStack directly into your project where the integrations will be loaded dynamically.

Reviewer 2: As we've already declared that the new architecture is the default.
```suggestion
While it is recommended to use the above-mentioned snippets to use the JavaScript SDK with your website, you can also use the [**NPM module**](https://www.npmjs.com/package/rudder-sdk-js) to package RudderStack directly into your project.
```

Author: updated
